tests/unit/rest.py

```diff
@@ -411,10 +493,12 @@ def test_validation_status(client, validation_setup, args):
     # HTTP method, so we hack around it by using getattr to get the right
     # function:
```

Comment:
I think this comment has been orphaned; it (a) probably was supposed to be in `_do_request`, but got separated from the code it was talking about somehow, and (b) is obsolete now that you've found client.open. Should probably just remove it.